### PR TITLE
formatting code for better readability

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Block/TimelineBlock.php [hunk 1]
```diff
@@ -97,8 +97,12 @@ public function buildEditForm(FormMapper $formMapper, BlockInterface $block)
     {
         $formMapper->add('settings', 'sonata_type_immutable_array', array(
             'keys' => array(
-                array('title', 'text', array('required' => false)),
-                array('max_per_page', 'integer', array('required' => true)),
+                array('title', 'text', array(
+                    'required' => false
```
Comment (Soullivaneuh) [inline Block/TimelineBlock.php]: `,` at EOL.

File: Block/TimelineBlock.php [hunk 2]
```diff
@@ -97,8 +97,12 @@ public function buildEditForm(FormMapper $formMapper, BlockInterface $block)
...
+                )),
+                array('max_per_page', 'integer', array(
+                    'required' => true
```
Comment (Soullivaneuh) [inline Block/TimelineBlock.php]: `,` at EOL.

File: Block/TimelineBlock.php (hunk 1 above)
Comment (OskarStark) [inline Block/TimelineBlock.php]: thanks done

File: Block/TimelineBlock.php (hunk 2 above)
Comment (OskarStark) [inline Block/TimelineBlock.php]: thanks done

**Pull request merged**